app/assets/javascripts/app/views/agent_ticket_create.jst.eco

```diff
@@ -37,6 +37,7 @@
 
             <div class="formset-inset">
               <div class="ticket-form-middle horizontal two-columns"></div>
+              <div class="ticket-form-organization horizontal two-columns"></div>
```

Comment:
Please don't introduce hard coded attribute logic into generic files.

Reply:
I added new Organization_id field for Ticket and it's place where this field puts.

Reply:
The field should use the existing generic hooks (`ticket-form-middle` in this case) as the other attributes do.